mlflow/tracing/provider.py

```diff
@@ -68,10 +68,14 @@ def _get_tracer(module_name: str):
     return tracer_provider.get_tracer(module_name)
 
 
-def _setup_tracer_provider():
+def _setup_tracer_provider(disabled=False):
     """
     Instantiate a tracer provider and set it as the global tracer provider.
     """
+    if disabled:
+        _force_set_otel_tracer_provider(trace.NoOpTracerProvider())
+        return
```

Comment:
Why do we need `disabled variable`? Can we directly call `_force_set_otel_tracer_provider(trace.NoOpTracerProvider())`?